### DISCUSSION TITLE
Serverless function time has increased significantly

### DISCUSSION THREAD

**Discussion by kristjanmar** [vercel/community#175] [Show & Tell]
Has anyone else seen a big increase in serverless function execution time recently?

The execution time has increased a lot for me in the past two weeks, despite function invocations not having increased and traffic being slightly lower than before. This is going to cost me hundreds per month in usage bills if it continues since Vercel charges a whopping $55 per 100GB-s.

Also, is there a way for me to see which serverless functions are contributing to these high usage times?

![Screenshot_9](https://user-images.githubusercontent.com/22234692/144015327-7e41bbb8-942f-45d5-b739-7a89ad47ce5c.png)

![Screenshot_10](https://user-images.githubusercontent.com/22234692/144015351-b235a7e0-457b-4ea4-a28d-3f1e0deb8b50.png)

**Comment by Pieparker** [comment]
Hey @kristjanmar, could you provide a deployment ID, commit SHA or anything else to help us identify your team and the site/function(s) in question? We'd be happy to take a look.

**Comment by leerob** [comment]
Wanted to follow up here and mention it should now be much easier to see what files, routes, ISR functions, etc are incurring usage.

https://vercel.com/changelog/enhanced-observability-on-the-usage-tab
